notebooks/summit/AuxTel/SITCOM-627/SITCOM-627_Measure_Astigmatism.ipynb

```diff
@@ -0,0 +1,828 @@
+{
```

Comment:
<p>you need to change this to LatissWEPAlign as well.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_isotuela/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>